src/commands/install.yml

```diff
@@ -9,12 +9,17 @@ parameters:
   # node
   node-version:
     type: string
-    default: ''
+    default: 'node'
     description: >
-      You may specify a full version tag ("13.11.0"), "latest", "current", "lts", or "stable".
+      Specificy the full version tag to install. To install the latest LTS version, set `lts` to true.
```

Comment:
If the user has "stable" or "current" in their config now, this would be a breaking change no? I think we'd have to consider this a major.

Reply:
Sure, can't hurt. Let's go ahead and mark this as major.